src/services/sessions.service.ts

```diff
@@ -74,12 +81,19 @@ export class SessionsService {
 
 			const logRecords = (response as any)?.log?.records;
 			const directRecords = response.records;
+			const outputs = response.records.map((record) => {
```

Comment:
Inside the component - the outputs should be only in the last record, and only if it's completed.